Orientation-API/Controllers/CustomersController.cs

```diff
@@ -56,6 +57,22 @@ public HttpResponseMessage Edit(int id, CustomerDto customer)
                     return Request.CreateErrorResponse(HttpStatusCode.InternalServerError, "Something went wrong, please try again later.");
             }
         }
+
+        [Route(""), HttpPost]
+        public HttpResponseMessage CreateCustomer(CustomerModel customer)
```

Comment:
I am wondering if we would want to change this to accepting a class of `CustomerDto` as opposed to `CustomerModel` since the model allows there to be an ID, and we don't necessarily want the creator to be able to set the ID since our DB is already doing that work. Doing that would then require mapping it to the `CustomerModel` after it has been accepted. If we are just wanting to get through the project we can leave it, but that was the only thought that I had when it came to the code. Awesome work!